tutorial/src/workspace/circuit.js

```diff
@@ -114,6 +115,67 @@ export function createCircuitAsciiArt(data_qubits, anc_qubit, with_time=true, fo
 
 
 
+/**
+ * Create the circuit as stim code snippet.
+ * 
+ * For simplicity, a single form of circuits is available:
+ * 
+ * [] "univ"
+ *    - Hadamard on the ancilla qubit
+ *    - both CNOT and CZ as 2q gates
+ *    - every CNOT/CZ is controlled by the ancilla and targets a data qubit
+ */
+export function createCircuitStimCode(data_qubits, anc_qubit) {
```

Comment:
This would be really awesome to have in the canonical `frontend` folder. Could you move new file?

`frontend/src/plaquettes/StimCircuit.js` would be good.

This way, we'll be able to download the circuit as a stim file, without having to invoke the backend server.

Reply:
I will add the same functionality to the frontend v1.

In general, this PR is about the development of the frontend v2.
We may want to discuss within the TQEC community if it makes sense to keep both (exploring different design choices may be a plus) or if we should focus on a single one (and, in this case, on which one).

Reply:
I added the function generating the stim code in `frontend/src/plaquettes/StimCircuitWriter.js`.

I could not confirm that the function behaves as desired (I am unfamiliar with the workflow in frontend v1). Please give a look.

Reply:
@giangiac You can add these features to the download button. Instead of invoking the backend, it can convert the circuit to `.stim` format and download the file.